### ISSUE TITLE
how to change trigger for host workflow fro on website startup on browser ( current default is manually )

### ISSUE THREAD

**Issue by vuduc231** [AutomaApp/automa#1288]

**Comment by rkabiri1988** [comment]
Stockoverflow

**Comment by WickedInvi** [comment]
Have you Synced the workflow on the machine where you are using it?

**Comment by rkabiri1988** [comment]
No! I' have try out automa with github emulator but i did not succeed!!!
I want an app, bot or even Java script code to help me easier because I am
amateur and my English language isn't good 😕
Please guide me more🙏

+989330019696

**Comment by vuduc231** [comment]
> Stockoverflow



> Have you Synced the workflow on the machine where you are using it?

I have 2 problems you have solutions for:
1. when I start the browser on the server side, I have a trigger that when the browser is opened will automatically run automa, I run about 10 to 20 browsers, then some browsers don't run automa by themselves.
2. on the server side I have sync on the cloud, when I want to update for other browsers, I have to go to each browser to update manually (must open automa to sync)
